spec/person_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './person'
+require './book'
+require './rental'
+
+describe Person do
+  let(:person) { Person.new(31, 'Enoh Collins') }
+
+  it 'should initialize correctly' do
+    expect(person.age).to eq(31)
+    expect(person.name).to eq('Enoh Collins')
+  end
+
+  it 'should correctly check age' do
+    expect(person.send(:of_age?)).to be true
+    person = Person.new(15, 'AbdulSami')
+    expect(person.send(:of_age?)).to be false
+  end
+
+  it 'should return correct name' do
+    expect(person.correct_name).to eq('Enoh Collins')
+  end
+
```

Comment:
- [ ] As your previous code reviewer mentioned please add the `test` case for the `default` parameterized constructor, where you just pass the `age` of the `person` class constructor and your result should pass for the below scenario.
`expect(person.name).to eq('Unknown')`. But I noticed that you also overlooked this `requirement`. Kindly implement the changes as I mentioned, in case something is not clear, ask in the comment sections by tagging @skyv26 with your doubt.